scripts/apply_patches.sh

```diff
@@ -48,9 +48,16 @@ replace_embedded_svg_icon \
 replace_embedded_svg_icon \
 	../resources/vaultwarden-password-manager-logo.svg \
 	./apps/web/src/app/layouts/password-manager-logo.ts
+replace_embedded_svg_icon \
+	../resources/src/images/logo.svg \
+	./libs/auth/src/angular/icons/bitwarden-logo.icon.ts
+replace_embedded_svg_icon \
+	../resources/vaultwarden-icon.svg \
+	./libs/auth/src/angular/icons/bitwarden-shield.icon.ts
 
 echo "Remove non-free bitwarden_license/ code"
 rm -rf ./bitwarden_license/
+rm -rf ./apps/web/src/app/tools/access-intelligence/
```

Comment:
Note, this should be resolved with next version, so we can remove this by then: https://github.com/bitwarden/clients/pull/12116

Reply:
How about this then?

```suggestion
[ -d ./apps/web/src/app/tools/access-intelligence/ ] && rm -rf ./apps/web/src/app/tools/access-intelligence/
```

Reply:
Yea, that seems like a good one.
Though, the patch needs to be fixed to, it will at least not break on this point then already.